test/02_BufferedToken.test.js

```diff
@@ -1,157 +0,0 @@
-const { BN, expectRevert } = require('@openzeppelin/test-helpers')
```

Comment:
Instead of removing this file, create a mock contract that gives you access to the internal functions and keep the tests. If we refactor functionality then we can remove them one by one when they are not needed.

Reply:
Yeah this seems worth at least trying...  To avoid scrapping useful tests.  Unless it becomes a big pain